flatm8s/src/pages/SignUp.js

```diff
@@ -44,14 +44,37 @@ class SignUp extends Component {
             .then((response) => response.json())
             .then((json) => {
                 console.log(json.userId)
+                let myHeaders = new Headers()
```

Comment:
I think with the amount of stuff going on in here it might make sense to start thinking about how you want to structure requests in the future. A possible option could be to have some sort of api service which is responsible for sending requests to routes. The input in the case would be the email + password

Reply:
i agree - especially when im doing multiple requests at a time, i may need some help there though

Reply:
Sweet as, totally something we can look at in the future :)